### DISCUSSION TITLE
Recipe Format: "Cooking for Engineers"

### DISCUSSION THREAD

**Discussion by dbrgn** [mealie-recipes/mealie#3920] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

For a few years, I've been writing down my recipes using the "Cooking for Engineers" format:

https://coolinfographics.com/blog/2010/4/26/cooking-for-engineersrecipe-infographics-and-interview.html
https://www.cookingforengineers.com/

Example:

![image](https://github.com/user-attachments/assets/58390280-c6e5-47bb-9cfc-5ef6ba65c46d)

This format has a few advantages, mainly that you can immediately see which steps can be parallelized, and which steps can't. Additionally, all ingredients are listed in the order of usage.

### Please provide a concise description of the feature that would resolve your issue.

It would be fantastic, if Mealie could offer a mode where recipes can be entered in this tabular flowchart-style format.

### Please consider and list out some caveats or tradeoffs made in your design decision

- It might be hard to implement
- Some people might prefer the conventional format
- Providing both formats in parallel would be an increased maintenance effort

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by tit1** [comment]
This is an interesting idea / format.. I do like the fact that it highlights what steps need to be done in parallel. I guess my main concern is that some of the steps details would be lost.
